__std__/plutus-apps/library/combine-haddock.nix

```diff
@@ -39,11 +38,13 @@ cell.library.pkgs.runCommand "combine-haddock"
   mkdir -p "$out/share/doc"
 
   for pkg in $hsdocsRec; do
-    files=($pkg/share/doc/*)
-    if [ ''${#files[@]} -gt 0 ]; then
-      cp -R ''${files[@]} "$out/share/doc"
+    if [ -d "$pkg/share/doc/ghc" ]; then
+      echo "Skipping GHC :'("
```

Comment:
Something weird is going on here.
The `share/doc/ghc` folder cannot be copied:
```
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/Debugger.html': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/DriverBkp.html': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/synopsis.png': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/linuwial.css': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/haddock-bundle.min.js': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/quick-jump.css': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/quick-jump.min.js': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/meta.json': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/ghc.txt': Permission denied
cp: cannot create directory '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/src': Permission denied
```
Any clues @michaelpj?